spec/lib/ruby_smb/smb1/tree_spec.rb

```diff
@@ -495,17 +495,17 @@
   describe '#open_pipe' do
```

Comment:
That would be great to have specs that make sure the backslash is added or removed, according to wether `open_pipe` or `open_file` is called. This also applies for SMBv2.